src/easypost.js

```diff
@@ -9,7 +9,7 @@ import Batch, { propTypes as batchPropTypes } from './resources/batch';
 import Brand, { propTypes as brandPropTypes } from './resources/brand';
 import CarrierAccount, { propTypes as carrierAccountPropTypes } from './resources/carrier_account';
 import CarrierType, { propTypes as carrierTypePropTypes } from './resources/carrier_type';
-import CreditCard, { propTypes as creditCardPropTypes } from './resources/credit_card';
+import Billing, { propTypes as BillingPropTypes } from './resources/billing';
```

Comment:
Issue: This is not correct as the actual propTypes got moved to `PaymentMethod`, please correct this.

Reply:
`paymentMethodPropTypes` is down below in line 19